<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DegenerateBases</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="sequenceResultsWidget">
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <row>
       <property name="text">
        <string>0</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Sequence</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDockWidget" name="sequenceDockWidget">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>250</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string>Sequence Manager</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="sequenceWidget">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="sequenceLabel">
       <property name="text">
        <string>Enter a sequence:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="sequenceEdit"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="sequenceButtonLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="sequenceButtonSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="loadButton">
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="executeButton">
         <property name="text">
          <string>Execute</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="informationDockWidget">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>250</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Sequence Information</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="informationWidget">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableWidget" name="informationTableWidget">
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="gridStyle">
        <enum>Qt::SolidLine</enum>
       </property>
       <property name="cornerButtonEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <row>
        <property name="text">
         <string>Sequence:</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Sequence's Length:</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Total Outcomes:</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Total Iterations:</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Permutation Time:</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Writing Time:</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>Value</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="DegenerateBases.qrc"/>
 </resources>
 <connections/>
</ui>
